### ISSUE TITLE
Blog posts being created as regular pages with Notice Undefined Index: ChildPages

### ISSUE THREAD

**Issue by andrewvt** [micmania1/silverstripe-blogger#53]
I just installed this module on a new install of SilverStripe in order to do some theming and when creating blog posts it is throwing this notice and then creating the post as a regular page.

This is what is in my composer.lock for the module:

"name": "micmania1/silverstripe-blog",
            "version": "dev-master",
            "source": {
                "type": "git",
                "url": "https://github.com/micmania1/silverstripe-blogger.git",
                "reference": "3c84269c54ae56ec8d747989a6e00d9809ffec15"
            },

**Comment by tractorcow** [comment]
Have you tried this on the 2.0.x-dev branch? I've had this before but I think it's fixed there.

**Comment by micmania1** [comment]
@andrewvt this has been fixed in #50. I'd advise against using anything other than the latest stable release which is currently 1.0.8.

**Comment by andrewvt** [comment]
I installed via composer using what was suggested here http://addons.silverstripe.org/add-ons/micmania1/silverstripe-blog which is "composer require micmania1/silverstripe-blog dev-master".  Here in the readme you say to use "composer require micmania1/silverstripe-blog:1.0.*".  That is probably why I ended up with a different version from the most stable.  

Now that I am on 1.0.8 as you suggested everything is working fine.  Just thought I would point out the discrepancy in the the two composer commands.  Thanks for the help.
